src/cli.zig

```diff
@@ -231,6 +232,12 @@ pub const Arguments = struct {
         Global.exit(0);
     }
 
+    fn printRevisionAndExit() noreturn {
+        @setCold(true);
+        Output.writer().writeAll(Global.package_json_version_with_revision ++ "\n") catch {};
```

Comment:
We should also add a test somewhere in `test/cli` that verifies:
* exit code is 0
* stdout starts with `bun --version`
* stdout matches basic semver regex